### PR TITLE
Add loc thank news

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/news/spanish-traditional-chinese-available.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (FeynmanZhou) [inline content/en/news/spanish-traditional-chinese-available.md:1]: Please add keywords and description here, ref: https://github.com/kubesphere/website/blob/master/content/en/news/kubesphere-geko-partnership.md

**Comment by FeynmanZhou** [comment]
[This News](https://github.com/kubesphere/website/blob/master/content/en/news/kubesphere-radore-partnership.md) is also missing keywords and description, please add them in.

File: content/en/news/spanish-traditional-chinese-available.md (hunk 1 above)
Comment (Sherlock113) [reply content/en/news/spanish-traditional-chinese-available.md:1]: keywords and description added

**Comment by Sherlock113** [comment]
> [This News](https://github.com/kubesphere/website/blob/master/content/en/news/kubesphere-radore-partnership.md) is also missing keywords and description, please add them in.

Added

File: content/en/news/spanish-traditional-chinese-available.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+keywords: KubeSphere, Localization, Community Contribution
+description: The web console of KubeSphere v3.0 now supports the localization of Spanish and Traditional Chinese.
+author: 'Feynman, Sherlock'
+---
+
+The web console of KubeSphere v3.0 now supports the localization of Spanish and Traditional Chinese. In User Settings, you can select the language of web console in the drop-down menu.
+
+![KubeSphere-language-setting](https://ap3.qingstor.com/kubesphere-website/docs/KubeSphere-language-setting.png)
+
+Here, we would like to thank Geko, our Spanish partner that provides the Spanish localization. This serves as a great start for the cooperation between KubeSphere and Geko as the two sides promise to further develop the Spanish community. With a brand-new Spanish web console, KubeSphere aims to create better user experiences for users around the world who prefer Spanish UI.
```
Comment (FeynmanZhou) [inline content/en/news/spanish-traditional-chinese-available.md:13]: I recommend you to add a hyperlink for the organization or people when you mention them in a post, they would be happy to see your introduction to their homepage or social media account.

Example: we would like to thank [Geko Cloud](https://geko.cloud/).

File: content/en/news/spanish-traditional-chinese-available.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+![KubeSphere-Spanish-console](https://ap3.qingstor.com/kubesphere-website/docs/KubeSphere-spanish.png)
+
+At the same time, we would also like to express our gratitude to Turtle Chang, who contributes to the localization of Traditional Chinese. Turtle is a Kubernetes architecture designer in Taiwan. As an active player in the community, Turtle has maintained close contact with us since the very beginning. In addition to localization, he will also help KubeSphere to further promote the local community in the future.
```
Comment (FeynmanZhou) [inline content/en/news/spanish-traditional-chinese-available.md:17]: Turtle Chang => [Turtle Chang](https://www.facebook.com/profile.php?id=100000179480251)

File: content/en/news/spanish-traditional-chinese-available.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+![kubesphere-traditional-chinese](https://ap3.qingstor.com/kubesphere-website/docs/Traditional-Chinese-KubeSphere.png)
+
+Community represents an essential part of KubeSphere. The localization provided by them is an excellent example of how community members can contribute to the open source platform so that everyone can benefit.
+
+If you are also interested in contributing to the localization of KubeSphere, please don’t hesitate to tell us and a [guide](https://github.com/kubesphere/community/blob/master/sig-docs/localization/how-to-localize-console.md) is available for reference.
```
Comment (FeynmanZhou) [inline content/en/news/spanish-traditional-chinese-available.md:23]: We expect more languages to be supported in KubeSphere Console. If you are ....

**Pull request merged**